snapcraft/hooks/configure

```diff
@@ -88,6 +89,7 @@ config="${SNAP_COMMON}/config"
     echo "lxcfs_loadavg=${lxcfs_loadavg:-"false"}"
     echo "lxcfs_pidfd=${lxcfs_pidfd:-"false"}"
     echo "lxcfs_cfs=${lxcfs_cfs:-"false"}"
+    echo "lxcfs_extra_opts=\"${lxcfs_extra_opts:-""}\""
```

Comment:
Why "extra_opts" rather than making it a dedicated debug option?

Reply:
I'm not sure of the convention with the LXD snap package, but we don't have so far an equivalent way of passing arbitrary flags to the LXD binary, so it may be this was restricted by design. CC @stgraber

Reply:
Because we may want to pass other libfuse options, like " -o attr_timeout".

Reply:
> I'm not sure of the convention with the LXD snap package, but we don't have so far an equivalent way of passing arbitrary flags to the LXD binary, so it may be this was restricted by design. CC @stgraber

understand. That's why code review is important :-) I'm newbie with snap, so...

Reply:
As I can see we have a very convenient trick for LXD https://github.com/lxc/lxd-pkg-snap/commit/8f340a6ce50cf295f48410a8be2b9ade74c6e651
I think it's worth adding an analogical thing for lxcfs. Then it will be okay just to have snap `lxcfs.debug` option, and if we need something extra, we can just override lxcfs binary with bash script with arbitrary options passed to the lxcfs binary.

Reply:
Yes the sideloading functionality is very useful. Although it does require that the binary is built within an equivalent environment as the snap provides (so shared libraries can link correctly).

Reply:
yep, it will work only if built from ubuntu 20 env. But that's better than nothing! ;-)